docs/Running/Publish.md

```diff
@@ -0,0 +1,83 @@
+# Publish & Deploy Apps
+
+Enarx is more than just a tool for running WebAssembly. It also allows you to publish a WebAssembly package to a remote package host, as well as securely run a published package in a local Enarx keep. The following sections will demonstrate this process, along with other useful commands for interacting with a package host.
```

Comment:
Why not just call it a Drawbridge registry with a reference to the project?
"package host" is very ambiguous, is that an SSH host? Is that an NFS share? Is that some custom service? People reading this should not be asking themselves this kind of questions

Reply:
I'm using "package host" specifically to avoid introducing the jargon "drawbridge", which is even less descriptive unless first explained. If we actually want to use "drawbridge" as a term then we can go ahead and define it here, but that was yet to be determined when this document was written.